### PR TITLE
Added timing information in recorded stroke

### PR DESCRIPTION
**Pull request by panglesd**
Sometimes, not only you need to record the stroke, but also the timing of the stroke, such as for instance [here](paulelliot.anglesdauriac.fr).

I added support for this, and updated `README.md` and the demo. Now, `stroke.points` contains an array of object with properties `point` (the coordinate) and `time` the time from the beginning of the stroke.

Anyway, thanks for your great tool, which I use often!

### CODE REVIEW

File: src/atrament.js [hunk 1]
```diff
@@ -206,6 +207,7 @@ module.exports = class Atrament extends AtramentEventTarget {
       this.dispatchEvent('strokerecorded', { stroke });
     }
     this.strokeMemory = [];
+    delete (this.strokeTimestamp);
```
Comment (jakubfiala) [inline src/atrament.js:210]: I'd say the `delete` here isn't needed, as we can just hold on to the last value and overwrite it when the next stroke begins - or am I missing an edge case?

**Review by jakubfiala — COMMENTED** [review]
This looks fantastic, thank you @panglesd for your contribution! I just have one tiny comment, otherwise all LGTM - once addressed I will merge the PR and release this change in Atrament 2.0 (since it does alter the API of the `stroke***` events).

File: src/atrament.js (hunk 1 above)
Comment (panglesd) [reply src/atrament.js:210]: Sure, the `delete` isn't needed! I added it for some consistency: if no stroke has started, no `strokeTimestamp` is defined. I can remove this line if you prefer!

Comment (jakubfiala) [reply src/atrament.js:210]: that actually sounds reasonable, thank you! will merge this now and release the new version

**Pull request merged**